packages/material-ui-lab/src/TreeItem/TreeItem.test.js

```diff
@@ -532,13 +532,8 @@ describe('<TreeItem />', () => {
           expect(screen.getByTestId('one')).to.have.attribute('aria-expanded', 'true');
 
           // move focus to node two
-          fireEvent.click(screen.getByText('two'));
-          fireEvent.click(screen.getByText('two'));
-          act(() => {
-            screen.getByRole('tree').focus();
-          });
-
-          expect(screen.getByTestId('two')).to.have.attribute('aria-expanded', 'false');
+          fireEvent.focusIn(screen.getByTestId('two'));
```

Comment:
```suggestion
          screen.getByTestId('two').focus()
```
probably needs `act`

Reply:
Thanks, I've squashed, un-squashed and then re-squashed this PR so many times I knew I'd forget something :P